components/Ripple/docs/ripple-migration.md

```diff
@@ -0,0 +1,55 @@
+Ink and Ripple provide similar APIs: a view (`MDCInkView`, `MDCRippleView`), and a touch controller (`MDCInkTouchController`, `MDCRippleTouchController`). 
+
+While Ripple and Ink’s implementations slightly vary, the public APIs are nearly identical. Furthermore, Ripple’s API does not produce additional side effects and can be used interchangeably.
+
+These two notions make the migration path from Ink to Ripple relatively simple.
+
+For guidance, these are the current naming differences that you need to pay attention to when migrating:
+
+**`MDCInkView` vs `MDCRippleView`:**
+
+|`MDCInkView`|`MDCRippleView`|
+|---|---|
+|`animationDelegate`|`rippleViewDelegate`|
+|`maxRippleRadius`|`maximumRadius`|
+|`startTouchBeganAtPoint:animated:completion:`|`beginRippleTouchDownAtPoint:animated:completion:`|
+|`startTouchEndAtPoint:animated:completion:`|`beginRippleTouchUpAnimated:completion:`|
+|`inkAnimationDidStart:inkView`|`rippleTouchDownAnimationDidBegin:rippleView`|
+|`inkAnimationDidEnd:inkView`|`rippleTouchUpAnimationDidEnd:rippleView`|
+
+*Other API differences are simply a regex replacement of: (1)Ink(2) to (1)Ripple(2) where (1) and (2) are [a-zA-Z]\*.*
+
+**`MDCInkTouchController` vs `MDCRippleTouchController`:**
+
+|`MDCInkTouchController`|`MDCRippleTouchController`|
+|---|---|
+|`defaultInkView`|`rippleView`|
+|`initWithView:` → `addInkView`|2 ways: <br> 1. `initWithView:`, <br> 2. `init` → `addRippleToView:`|
```

Comment:
I'm not sure I understand this particular step. Is the before two lines of code that need to be updated to either one or the other of the after? If so it may be easier for the reader to provide a single recommended replacement, with caveats outlined outside of this table.